lib/cocoaseeds/core.rb

```diff
@@ -564,18 +577,39 @@ def configure_phase
             phase.files.each do |build_file|
               phase.files.delete(build_file) if build_file.file_ref == file
             end
+            resource_phase.files.each do |build_file|
+              resource_phase.files.delete(build_file) if build_file.file_ref == file
+            end
           end
 
           addings.each do |seed_names|
             next if file.name.end_with? ".h"
             next if not seed_names.include?(file.parent.name)
             uuid = Xcodeproj::uuid_with_name "#{target.name}:#{file.name}"
-            phase.add_file_reference_with_uuid(file, uuid, true)
+            # Treat a file as resource file unless confirm it can be compiled.
+            if self.valid_source_file?(file)
+              phase.add_file_reference_with_uuid(file, uuid, true)
+            else
+              resource_phase.add_file_reference(file)
```

Comment:
How about making `add_file_reference_with_uuid()` function? It uses file name to create UUID in order to not create file diffs.